v2/errors.go

```diff
@@ -70,11 +70,11 @@ func IsConflictError(err error) bool {
 }
 
 const (
-	asyncErrorMessage     = "AsyncRequired"
-	asyncErrorDescription = "This service plan requires client support for asynchronous service operations."
+	AsyncErrorMessage     = "AsyncRequired"
```

Comment:
I like the intent here, but I think some exported methods from the fake package that just give you canned versions of these errors is probably more useful - revise to that?